.github/workflows/publish-stage.yaml

```diff
@@ -3,9 +3,7 @@ name: publish-stage
 on:
   push:
     branches:
-      - "main"
-    tags-ignore:
-      - "*"
+      - "unsupported/0.25"
```

Comment:
I'm more confident about this trigger than the production one. 

I removed `tags-ignore` because I don't know what effect it has? I should probably research it instead of blindly removing.

Reply:
ah, I bet the tags-ignore is there so that it doesn't re-deploy to staging when we do a production deploy (via tagging the branch). 

I'm going to put the tags-ignore back!